src/unit-taxonomy-tags-drawer/TaxonomyTagsCollapsible.jsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import {
+  Badge,
+  Collapsible,
+  SelectableBox,
+  Button,
+  ModalPopup,
+  useToggle,
+} from '@edx/paragon';
+import { injectIntl } from '@edx/frontend-platform/i18n';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import './TaxonomyTagsCollapsible.scss';
+
+import TaxonomyTagsDropDownSelector from './TaxonomyTagsDropDownSelector';
+import { useTaxonomyTagsDataResponse, useIsTaxonomyTagsDataLoaded } from './api/hooks/selectors';
+
+import TagBubble from './TagBubble';
+
+const TaxonomyTagsCollapsible = ({ taxonomyAndTagsData }) => {
+  const {
+    id, name, unitTags,
+  } = taxonomyAndTagsData;
+
+  const [isOpen, open, close] = useToggle(false);
+  const [target, setTarget] = React.useState(null);
+
+  const useTaxonomyTagsData = (taxonomyId, fullPathProvided) => {
+    const taxonomyTagsData = useTaxonomyTagsDataResponse(taxonomyId, fullPathProvided);
+    const isTaxonomyTagsLoaded = useIsTaxonomyTagsDataLoaded(taxonomyId, fullPathProvided);
+    return { taxonomyTagsData, isTaxonomyTagsLoaded };
+  };
+
+  const { taxonomyTagsData, isTaxonomyTagsLoaded } = useTaxonomyTagsData(id);
+
+  return (
+    <div className="taxonomy-tags-collapsible-div">
+      <Collapsible title={name} styling="card-lg" className="taxonomy-tags-collapsible">
+        <div key={id}>
+          {
+            unitTags.map((unitTag) => {
+              let level = -1; // Starting from -1 since we increment at beginning of loop
+              return unitTag.lineage.map((lineageTag) => {
+                level += 1;
+                return (<TagBubble key={`tag-${lineageTag}`} value={lineageTag} implicit={lineageTag !== unitTag.value} level={level} />);
+              });
+            })
+          }
+        </div>
+
+        <div className="taxonomy-tags-selector-menu">
+          <Button
+            ref={setTarget}
+            variant="outline-primary"
+            onClick={open}
+          >
+            Add tags
```

Comment:
Don't forget i18n - this should be `<FormattedMessage {...messages.addTagsButton} />`